doc/testing-virtual-machine.md

```diff
@@ -0,0 +1,33 @@
+## E2E conformance test
+
+It's possible to use QEMU to test the SR-IOV operator on a virtual kubernetes/openshift cluster.
+Using the IGB model network driver allow to create virtual functions on the virtual system
+
+## How to test
+
+First you will need to enable the `DEV_MOD` via the operator environment variable.
+Second step is to add the intel virtual nic to the supported nics configmap.
+
+```
+Intel_ixgbe_82576: 8086 10c9 10ca
```

Comment:
Can we permanently add this NIC to the config-map? What are the drawbacks?

Reply:
for example right now there are the two bugs on the nic so I will not like to add a nic that doesn't pass our tests.
maybe after the kernel fix the driver we can think on adding the nic, and asking Intel if they are fine supporting it :)

/cc @Eoghan1232